docs/src/meshes/p4est_mesh.md

```diff
@@ -3,7 +3,7 @@
 The [`P4estMesh`](@ref) is an unstructured, curvilinear, nonconforming
 mesh type for quadrilateral (2D) and hexahedral (3D) cells.
 It supports quadtree/octree-based adaptive mesh refinement (AMR) via
-the C library [p4est](https://github.com/cburstedde/p4est). See
+the C library [`p4est`](https://github.com/cburstedde/p4est). See
```

Comment:
```suggestion
the C library [p4est](https://github.com/cburstedde/p4est). See
```
IIRC, we typically do not render other software packages in code font.

Reply:
OK, I see that you consistently put p4est in backticks throughout this PR. I don't have a strong opinion here, so if this is desired, you can just leave it as it is done in here.

Reply:
I think we did it for `p4est` since they use that style in their paper.

Reply:
OK, then never mind my comment. I'm very late to the party, anyways 😬